### ISSUE TITLE
Regression in Screenshots.getLastScreenshot()

### ISSUE THREAD

**Issue by tdelphi** [selenide/selenide#880]
## Regression bug
Screenshots.getLastScreenshot() returns null after upgrade to Selenide 5.x

## Environment
* **Selenide Version:** 5.0.1
* **Junit Version:** 4.x

## Details
Declare test with
```
    @Rule
    public ScreenShooter makeScreenshotOnFailure = ScreenShooter.failedTests();
```
After test failed, try to get last screenshot via Screenshots.getLastScreenshot()

### What is expected
Path to last screenshot is returned (as it worked in Selenide 4.x)

### What happened
Screenshots.getLastScreenshot() returns null

### Workaround
Call ScreenShotLaboratory.getInstance().getLastScreenshot()

### Suggestion
* either fix, or remove Screenshots.java

**Comment by darklynx** [comment]
`Screenshots` class is used in [ScreenShooter](https://github.com/selenide/selenide/blob/master/statics/src/main/java/com/codeborne/selenide/testng/ScreenShooter.java)

This might cause some concurrency issues when tests are running with TestNG and result in lost screenshots mentioned in the issue #814

**Comment by asolntsev** [comment]
@tdelphi Sorry, I cannot reproduce the problem. 
Can you specify at which moment do you call `Screenshots.getLastScreenshot()`?
You said: "After test failed", but I don't understand. After test failed, your test code cannot be executed anymore: test is already failed. 

Or do you call `Screenshots.getLastScreenshot()` from your custom rule/listener/extension? 
Then it's important how do you declare your rules (in which order are they actually executed).

In other words, could you give us a code that reproduces the problem?

**Comment by asolntsev** [comment]
@tdelphi Can you check if this problem was fixed in Selenide 5.7.0 (by pull request #1052)?
